### PR TITLE
feat: Add OneKey Wallet support

### PR DESCRIPTION
**Pull request by AndyJiangPro**
Why do I submit this merge request?

PancakeSwap is the most popular decentralized platform. [OneKey wallet](https://onekey.so/) is a secure, user-friendly crypto wallet. It provides both hardware and software crypto wallet solutions. It has millions of users and an excellent community reputation.

Adding OneKey wallet support for PancakeSwap can benefit millions of PancakeSwap and OneKey Wallet users.

Unit test is good and I have tested my changes thoroughly(covering different kinds of edge cases) and it works very well.

### CODE REVIEW

**Comment by AndyJiangPro** [comment]
https://web-p2syiivnc.pancake.run/
Tested it and it is working properly.

File: src/config/wallet.ts [hunk 1]
```diff
@@ -138,4 +139,16 @@ export const wallets: WalletConfig<ConnectorNames>[] = [
       return typeof window !== 'undefined' && Boolean(window.ethereum?.isBlocto) ? 0 : 999
     },
   },
+  {
+    title: 'OneKeyWallet',
+    icon: OneKeyWalletIcon,
+    // @ts-ignore
+    installed: typeof window !== 'undefined' && Boolean(window.$onekey?.solana.isOneKey),
```
Comment (loatheb) [inline src/config/wallet.ts:146]: Thanks! You can use `window.$onekey.ethereum.isOneKey` to detect is OneKey in EVM.

Comment (AndyJiangPro) [reply src/config/wallet.ts]: Updated
Thanks

**Review by loatheb — APPROVED** [review]

**Comment by AndyJiangPro** [comment]
Hey，
@chef-jojo @ChefMomota @chefilip @Chef-Yogi 
Could you please take a look. The change is quite simple & straightforward. 
Thank you very much indeed.

**Comment by chef-jojo** [comment]
we decided not to add more wallets icon there 
I refactored the wallet modal https://github.com/pancakeswap/pancake-frontend/pull/4795
An injected wallet icon for any injected compatible wallets as fallback

to list the wallet image, you can go through our bd process, or you can leave us how to contact you for business partnership

**Comment by chef-jojo** [comment]
please contact pancakeswapbd on tg

**Comment by AndyJiangPro** [comment]
> we decided not to add more wallets icon there I refactored the wallet modal #4795 An injected wallet icon for any injected compatible wallets as fallback
> 
> to list the wallet image, you can go through our bd process, or you can leave us how to contact you for business partnership

Thanks
I will rebase & reafctor. and update this pull request.
